open_ce/inputs.py

```diff
@@ -410,6 +422,8 @@ def parse_args(parser, arg_strings=None):
 
     _check_and_create_fips_packages(args, arg_strings)
 
+    _check_cuda_versions(args)
+
```

Comment:
This call should go before `_check_and_create_fips_packages` so that python version change is also applied to fips packages (openssl-env.yaml).